### PR TITLE
feat: art quiz icon animation

### PR DESCRIPTION
**Pull request by rajsam003**
This PR resolves [GRO-1266] 

https://user-images.githubusercontent.com/17421923/221200910-5c63cf64-e132-457e-8e37-946b21ece901.mov

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[GRO-1266]: https://artsyproduct.atlassian.net/browse/GRO-1266?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by MounirDhahri — DISMISSED** [review]
Nice! thanks Sam

File: src/app/Scenes/ArtQuiz/ArtQuizButton.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  })
+
+  const animatedStyle = useAnimatedStyle(() => {
+    const scale = interpolate(progress.value, [0, 1], [1, 1.25], Extrapolate.CLAMP)
+    return {
+      transform: [{ scale }],
+    }
+  })
+
+  const hanldeOnPressIn = () => {
```
Comment (damassi) [inline src/app/Scenes/ArtQuiz/ArtQuizButton.tsx:35]:
```suggestion
  const handleOnPressIn = () => {
```

File: src/app/Scenes/ArtQuiz/ArtQuizButton.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    setIsAnimating(true)
+    pressed.value = true
+  }
+
+  const hanldeOnPressOut = () => {
```
Comment (damassi) [inline src/app/Scenes/ArtQuiz/ArtQuizButton.tsx:40]:
```suggestion
  const handleOnPressOut = () => {
```

File: src/app/Scenes/ArtQuiz/ArtQuizButton.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    setIsAnimating(false)
+    onPress()
+    pressed.value = false
+  }
+
+  return (
+    <Touchable onPressIn={hanldeOnPressIn} onPressOut={hanldeOnPressOut}>
```
Comment (TMcMeans) [inline src/app/Scenes/ArtQuiz/ArtQuizButton.tsx:47]: also another little nitpick here: 
```suggestion
    <Touchable onPressIn={handleOnPressIn} onPressOut={handleOnPressOut}>
```

**Review by TMcMeans — DISMISSED** [review]
LGTM 👍🏾 🥇

the buttons look great :) is there also going to be another follow up PR to handle the swipe function on the artwork cards to match the buttons?

**Comment by rajsam003** [comment]
> is there also going to be another follow-up PR to handle the swipe function on the artwork cards to match the buttons?

Yes. I am working on it. I will create a new PR for that. Feel free to merge this one please.

**Pull request merged**